.github/workflows/msvc.yml

```diff
@@ -0,0 +1,66 @@
+name: build
```

Comment:
@rrrooommmaaa you should change the `name:` value to `windows-msvc`? Better also to rename the file to `windows-mdvc.yml` for consistency (when we fetch the GHA badge).

Reply:
Done